src/DynamoCoreWpf/UI/Themes/Modern/Ports.xaml

```diff
@@ -72,6 +72,11 @@
                     </dynui:DynamoToolTip>
                 </Grid.ToolTip>
 
+                <!--Bind NodeAutoComplete to double left click-->
+                <Grid.InputBindings>
+                    <MouseBinding MouseAction="LeftDoubleClick" Command="{Binding Path=NodeAutoCompleteCommand}"></MouseBinding>
+                </Grid.InputBindings>
```

Comment:
Bind NodeAutoComplete to double left click here using `Grid.InputBindings` since Grid does not expose double left click event directly